content/W2017/W17-event-beckwith-talk.md

```diff
@@ -0,0 +1,29 @@
+Title: Tech Talk: Hotspot Virtual Machine's Execution Engine by Monica Beckwith
+Date: 2017-03-14
+Category: Events
+Tags: events, academic
+Slug: beckwith-tech-talk-w17
+Author: Monica Xu
+Summary: Come listen about the amazing work Monica Beckwith is doing!
+
+Monica Beckwith has worked with the Java Virtual Machine 
+for more than a decade not just optimizing the JVM heuristics,
+but also improving the Just-in-time (JIT) code quality for 
+various processor architectures as well as working with the 
+garbage collectors and improving garbage collection for server systems.
+
+During this talk, Monica will cover a few JIT and Runtime 
+optimizations and she will dive into the HotSpot garbage 
+collection and provide an overview of the various garbage
+collectors available in HotSpot.
+
+Note: Please claim your free admission ticket [here]
+(https://www.eventbrite.com/e/monica-beckwith-hotspot-
+virtual-machines-execution-engine-talk-tickets-32482143967)
+
+## Event Details ##
+
++ **Who:** Everyone
++ **What:** Tech Talk
```

Comment:
Again, have a look at the Leigh Honeywell post for a good idea of how to use this section 😄